Assets/MixedRealityToolkit/UX/Scripts/AppBar/AppBarButton.cs

```diff
@@ -39,6 +40,8 @@ public void Initialize(AppBar newParentToolBar, AppBar.ButtonTemplate newTemplat
             text = GetComponent<CompoundButtonText>();
             text.Text = template.Text;
             icon = GetComponent<CompoundButtonIcon>();
+            highlightMesh = cButton.transform.Find("FrontPlate").GetComponent<MeshRenderer>();
```

Comment:
Where in the hierarchy is the `FrontPlate` in relation to the GameObject this `AppBarButton` is attached to?

Reply:
AppBarButton uses HolographicButton prefab as a template, and it looks like this:
![2018-04-02 09_29_35-unity 2017 2 1p2 64bit - boundingboxgizmoexample unity - mrtk-original - unive](https://user-images.githubusercontent.com/13754172/38204652-a1a13866-3658-11e8-8872-f8f225ba7fbd.png)

Reply:
So it's dynamicly generated?

Or do we have a game object in the scene to reference?

Reply:
Yes, it is dynamically generated.

Reply:
I don't like that we're using `cButton.transform.Find("FrontPlate")`.  Is there a way we can do this using a `GameObject` reference instead of a string lookup?

Reply:
Wouldn't `GetComponent<CompoundButtonMesh>().Renderer` work?